sdk/servicebus/service-bus/test/stress/stressTestsBase.ts

```diff
@@ -59,9 +77,6 @@ export class SBStressTestsBase {
     process.env.SERVICEBUS_CONNECTION_STRING!
   );
   queueName!: string;
-  reportFileName: string;
-  errorsFileName: string;
-  messagesReportFileName: string;
```

Comment:
Bye bye, file storage!

Reply:
Yes, _but_ I did look at how we can plug into the appinsights pipeline so if we want to bring it back we can do that!